lib/import.php

```diff
@@ -197,6 +197,16 @@ protected function blob_to_post( WordPress_GitHub_Sync_Blob $blob ) {
 				$args['post_date_gmt'] = get_gmt_from_date( $meta['post_date'] );
 				unset( $meta['post_date'] );
 			}
+			
+			if ( array_key_exists( 'post_excerpt', $meta ) ) {
+				$args['post_excerpt'] = $meta['post_excerpt'];
+				unset( $meta['post_excerpt'] );
+			}
```

Comment:
I think we should just use `excerpt` here rather than `post_excerpt`, similar to `author` instead of `post_author`.